### ISSUE TITLE
Initial image for two secs *then* white screen

### ISSUE THREAD

**Issue by d-a-v** [DS-Homebrew/TWiLightMenu#2437]
**Flashcard or Exploits?** 

Original R4 (noSDHC)

**Base Console:** 

DS Lite

**TWiLight Menu++ Version:** 

Tried with: 27.5.1 and 27.4.0

Same issue

**TWiLight Menu++ Theme:** 

First installation, all defaults

### Expected behavior

Trying for the first time.

### Actual behavior

First image appears, then two seconds after, white screen.
One time, after waiting, the initial image came back, then vanished again two seconds after.

### Steps to reproduce

I followed [initial installation](https://wiki.ds-homebrew.com/twilightmenu/installing-flashcard), with Original R4 autoboot.
I also have "wood autoboot", and starting TWiLightMenu from there gives the exact same behaviour (first image for two secs, then white screen)

### Notes

I tried on two different DSLite
I tried two versions of TWiLightMenu (overwriting everything and removing config files) 27.5.1 and 27.4.0 -> same behaviour.
From the other wooden autoboot, I could start CPC emulator and play Cauldron II, Starquake, and GBA Doom II (though I should try with another emulator)

**Comment by RocketRobz** [comment]
When you say "initial image", do you mean the TWLMenu++ and Rocket Robz logos?

**Comment by d-a-v** [comment]
I mean something like this image on top screen



![proxy-image](https://github.com/DS-Homebrew/TWiLightMenu/assets/4800356/88aa0595-3133-4c24-b826-f8ab23188596)

**Comment by RocketRobz** [comment]
> I mean something like this image on top screen
> 
> ![proxy-image](https://private-user-images.githubusercontent.com/4800356/345210351-88aa0595-3133-4c24-b826-f8ab23188596.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk5NTcwNzcsIm5iZiI6MTcxOTk1Njc3NywicGF0aCI6Ii80ODAwMzU2LzM0NTIxMDM1MS04OGFhMDU5NS0zMTMzLTRjMjQtYjgyNi1mOGFiMjMxODg1OTYuanBlZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzAyVDIxNDYxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRhMWU3OWUwNmRiYjg2YzhiMTZkNTg3NzgxMThmZTliZjg4NGY5NGYwMTY0MWMzNDYyYTdjOWI3NDYwNzAyNzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jjGn1HWXmWPz7WEvL9fb4SC0r_K_Hk8SuVj-nNOxQcA)

That's a "yes" to my question then. Hold SELECT on that screen, and change the theme to either `R4 Original` or `Wood UI`.
Also, where did you find this image? Because that's been updated a while back.

**Comment by d-a-v** [comment]
Well I have to say that I may have been wrong.
I have to wait for 20 seconds without touching anything then things start by themselves !

Also, I did hold select on that screen and I indeed got into the config screen.

Sorry for the noise, I am now going to put 27.5.1 back and discover.
Thanks for the work, it looks very clean.

Both versions 27.5.1 and 27.4.0 are taken from the release button (version > assets > *_flashcard) on the right side of the "code" tab.
